### PR TITLE
Remove "tests.resources" from mint_fake_ocean & add wallets depending test keys for marketplace quickstart 

### LINKED ISSUE #324: Marketplace quickstart crashes when minting fake OCEAN
When I try to reproduce the market quickstart, I got stuck on minting part. After the installation of the ocean-lib, I have noticed that minting process crashes because of "tests.resources" module which said that is not found and that means that the wallets such as `ganache wallet` or `publisher_ocean_instance` could not be used.

### PR DESCRIPTION
**Pull request by mariacarmina**
Fixes #324 .

Changes proposed in this PR:

- tested & updated marketplace-flow README;
- fixed some modifications in `mint_fake_ocean` regarding wallets.

### CODE REVIEW

**Comment by mariacarmina** [comment]
Note: To actually work, we need a pypi release for that.

**Review by calina-c — DISMISSED** [review]
Approving, but it should also be seen by @trentmc.
Reasons for this change: the marketplace readme relied on files inside "tests" folder, which is outside the scope of the pypi release. It is corrected now.

File: READMEs/marketplace-flow.md [hunk 1]
```diff
@@ -101,6 +101,7 @@ In the work console:
 #set private keys of two accounts
 export TEST_PRIVATE_KEY1=0xbbfbee4961061d506ffbb11dfea64eba16355cbf1d9c29613126ba7fec0aed5d
 export TEST_PRIVATE_KEY2=0x804365e293b9fab9bd11bddd39082396d56d30779efbb3ffb0a6089027902c4a
+export FACTORY_DEPLOYER_PRIVATE_KEY=0xc594c6e5def4bab63ac29eed19a134c130388f74f019bc74b8f4389df2837a58
```
Comment (trentmc) [inline READMEs/marketplace-flow.md:104]: Why is this added? Is it deploying a new factory? It shouldn't be - it should be using the contracts already deployed, with addresses specified in addresses.json.

**Comment by mariacarmina** [comment]
The reason why diff-coverage test is failling is because I cannot add an unit test for this type of change.

**Review by trentmc — CHANGES_REQUESTED** [review]
This PR doesn't seem to have anything to do with the issue of https://github.com/oceanprotocol/provider/issues/54 "Provider: ocean.py marketplace quickstart hangs when downloading >1 files".

Rather, it seems to be fixing some apparent problem in marketplace quickstart. 

That's fine. But then first you need to report the problem as a new issue in ocean.py. And point to the issue in this PR (change the title, etc). Then use this PR to point the the bug that it's actually fixing. And then describe how the changes fix the bug.

Finally, I don't see evidence that the issue of https://github.com/oceanprotocol/provider/issues/54 "Provider: ocean.py marketplace quickstart hangs when downloading >1 files" was actually reproduced, let alone in tests going forward. Please reproduce it by writing a unit/system test that has >1 file service for the same ERC20 token.  (Like it says in the issue description: "write a unit or system test on >1 file.")

File: READMEs/marketplace-flow.md (hunk 1 above)
Comment (mariacarmina) [reply READMEs/marketplace-flow.md:104]: I saw that the ganache wallet uses this key, so that's why I have exported it before. Should I replace it?

Comment (trentmc) [reply READMEs/marketplace-flow.md:104]: I can't comment more on this PR until I see a github issue reporting what this PR is trying to fix.

Comment (trentmc) [reply READMEs/marketplace-flow.md:104]: Thanks for creating the github issue, and linking to it here. +1

It doesn't seem logical that someone using a quickstart should have to point to the private key of some deployer. The only private keys a quickstart should really have are those of the user. So, something's up. You'll need to get to the bottom of this.

Comment (trentmc) [reply READMEs/marketplace-flow.md:104]: I thought more about it. This is really here in order to mint fake ocean. Alright, fair enough.

But you need to make it more clear why that private key is being exported. For example, this part of the README could be like:
```console
#set private keys of two accounts
export TEST_PRIVATE_KEY1=0xbbfbee4961061d506ffbb11dfea64eba16355cbf1d9c29613126ba7fec0aed5d
export TEST_PRIVATE_KEY2=0x804365e293b9fab9bd11bddd39082396d56d30779efbb3ffb0a6089027902c4a

#needed to mint fake OCEAN
export FACTORY_DEPLOYER_PRIVATE_KEY=0xc594c6e5def4bab63ac29eed19a134c130388f74f019bc74b8f4389df2837a58

#start python
python
```

**Review by trentmc — CHANGES_REQUESTED** [review]
Small tweak requested, mentioned in a comment. Then we're good to go.

File: READMEs/marketplace-flow.md (hunk 1 above)
Comment (mariacarmina) [reply READMEs/marketplace-flow.md:104]: Ok, will modify right now.

**Review by trentmc — APPROVED** [review]

**Comment by trentmc** [comment]
Approved from my side now too.

**Comment by calina-c** [comment]
> This PR doesn't seem to have anything to do with the issue of [oceanprotocol/provider#54](https://github.com/oceanprotocol/provider/issues/54) "Provider: ocean.py marketplace quickstart hangs when downloading >1 files".
> 
> Rather, it seems to be fixing some apparent problem in marketplace quickstart.
> 
> That's fine. But then first you need to report the problem as a new issue in ocean.py. And point to the issue in this PR (change the title, etc). Then use this PR to point the the bug that it's actually fixing. And then describe how the changes fix the bug.
> 
> Finally, I don't see evidence that the issue of [oceanprotocol/provider#54](https://github.com/oceanprotocol/provider/issues/54) "Provider: ocean.py marketplace quickstart hangs when downloading >1 files" was actually reproduced, let alone in tests going forward. Please reproduce it by writing a unit/system test that has >1 file service for the same ERC20 token. (Like it says in the issue description: "write a unit or system test on >1 file.")

I agree to adding the extra ticket. But I also don't know what else Maria can test. We can't have unit tests on the marketplace readme since it implies the market components. And we would basically do marketplace_flow again. I also understood the scope of this ticket was to test the multiple files case using the README and check if it reproduces. @mariacarmina You did this, right?

We might be looking at a fluke in provider that doesn't reproduce at all, or a speed issue, or an issue related to asset availability. (i.e. the hosting for the asset or assets was compromised temporarily). The question is how much effort should we put into this by this point.

**Comment by mariacarmina** [comment]
> > This PR doesn't seem to have anything to do with the issue of [oceanprotocol/provider#54](https://github.com/oceanprotocol/provider/issues/54) "Provider: ocean.py marketplace quickstart hangs when downloading >1 files".
> > Rather, it seems to be fixing some apparent problem in marketplace quickstart.
> > That's fine. But then first you need to report the problem as a new issue in ocean.py. And point to the issue in this PR (change the title, etc). Then use this PR to point the the bug that it's actually fixing. And then describe how the changes fix the bug.
> > Finally, I don't see evidence that the issue of [oceanprotocol/provider#54](https://github.com/oceanprotocol/provider/issues/54) "Provider: ocean.py marketplace quickstart hangs when downloading >1 files" was actually reproduced, let alone in tests going forward. Please reproduce it by writing a unit/system test that has >1 file service for the same ERC20 token. (Like it says in the issue description: "write a unit or system test on >1 file.")
> 
> I agree to adding the extra ticket. But I also don't know what else Maria can test. We can't have unit tests on the marketplace readme since it implies the market components. And we would basically do marketplace_flow again. I also understood the scope of this ticket was to test the multiple files case using the README and check if it reproduces. @mariacarmina You did this, right?
> 
> We might be looking at a fluke in provider that doesn't reproduce at all, or a speed issue, or an issue related to asset availability. (i.e. the hosting for the asset or assets was compromised temporarily). The question is how much effort should we put into this by this point.

Yep, I did.

**Comment by trentmc** [comment]
> But I also don't know what else Maria can test. 

It's: a unit/system test that has >1 file service for the same ERC20 token. 

(This is not in our current marketplace README, nor does it need to be there. We simply need to test this)

**Comment by calina-c** [comment]
> > But I also don't know what else Maria can test.
> 
> It's: a unit/system test that has >1 file service for the same ERC20 token.
> 
> (This is not in our current marketplace README, nor does it need to be there. We simply need to test this)
> 
> BTW that work is outside the scope of this PR, it's for [oceanprotocol/provider#54](https://github.com/oceanprotocol/provider/issues/54) . We should move our discussion either to that ticket or to a PR/branch working on addressing that ticket.

Got it. So we'll add a new test in the marketplace flow.
Btw ref CodeClimate checks, I think it's broken today. It didn't analyse some provider branches either.
